src/OpenSSL/SSL.py

```diff
@@ -1113,11 +1131,13 @@ def _raise_passphrase_exception(self):
 
         _raise_current_error()
 
-    def use_privatekey_file(self, keyfile, filetype=_UNSPECIFIED):
+    def use_privatekey_file(
+        self, keyfile: _StrOrBytesPath, filetype=_UNSPECIFIED
```

Comment:
```suggestion
        self, keyfile: _StrOrBytesPath, filetype: int=_UNSPECIFIED
```

Reply:
Unfortunately the UNSPECIFIED sentinel is an object. Is there any reason we are not using `None` here?

Reply:
I don't see a reason, no.

Reply:
OK, so `filetype: int = FILETYPE_PEM` it is!